src/newcomponents/attach/index.js

```diff
@@ -0,0 +1,179 @@
+// export default function Attach() {
```

Comment:
do we need this file if all the code is commented?

Reply:
In the future, we will use this logic as part of a separate component